java/openjdk20-openj9/Portfile

```diff
@@ -1,97 +1,12 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem       1.0
+PortSystem              1.0
 
-name             openjdk20-openj9
-categories       java devel
-maintainers      {breun.nl:nils @breun} openmaintainer
-platforms        {darwin any}
-# This port uses prebuilt binaries; 'NoMirror' makes sure MacPorts doesn't mirror/distribute these third-party binaries
-license          GPL-2 NoMirror
-# This port uses prebuilt binaries for a particular architecture; they are not universal binaries
-universal_variant no
+# This port can be removed on 2024-11-26.
+replaced_by             openjdk21-openj9
+PortGroup               obsolete 1.0
```

Comment:
`PortGroup` entries should be at the top of the Portfile directly under `PortSystem`.

Reply:
Why does the order of lines matter? I took the example from [a Portfile recipe](https://trac.macports.org/wiki/PortfileRecipes#replaced-by) and `port lint --nitpick` doesn't complain.

Reply:
The order mostly matters because people doing reviews are used to looking in certain parts of the files.

Reply:
I've adjusted the PR, but if wiki examples and the official linting tool don't are fine with any ordering, then I think that making `port lint` / `port lint --nitpick` make a note of this would be much more efficient run than reviewers manually requesting a particular undocumented order.

Reply:
@breun I think that might be a good thing. Right now, not a lot of work is going into port lint, but there are a bunch of things that could be improved. There are times when it gives warnings that we know shouldn't be given, other things it should warn about (like excessively long lines that aren't associated with the top line or checksums) etc. If you wanted to have a whack at it, I think people would be grateful.